README.md

```diff
@@ -0,0 +1,136 @@
+## Empathy
+
+![image](https://user-images.githubusercontent.com/240083/37926477-4c464bde-30ec-11e8-8a0f-1d2b4b7521d0.png)
+Empathy is a minimalist tool that generates a folder of "asset" dependencies for
+an NPM package. It is designed in the spirit of the 
+[`npm/npm#npm-asset`](https://github.com/npm/npm/tree/npm-asset) work being done
+by NPM.
+
+Asset dependencies are "Web Ready" NPM packages whose specifiers have been
+adjusted to point to their dependencies at the appropriate paths in the local
+file tree.
+
+Empathy can also transform local source files to use the correct "name" 
+specifiers when it comes time to publish those sources to NPM.
+
+### Usage
+
+Add `empathy` as a dev dependency of your package:
+
+```
+$ npm i --save-dev @0xcda7a/empathy
+```
+
+Then include your assets as dependencies in your `package.json`. For example, 
+this partial manifest includes an asset dependency on Polymer v3.0.0-pre:
+
+```js
+{
+  "name": "cool-dogs-package",
+  "version": "1.0.0",
+  // ...
+  "dependencies": {
+    "@polymer/polymer": "v3.0.0-pre.12",
+    // and other dependencies...
+  },
+  // ...
+}
+```
+
+#### Installing assets from NPM
+
+The easiest way to install assets from NPM with `empathy` is to run the 
+`install` command:
+
+```
+$ empathy install
+```
+
+You should see console output that shows progress generating your asset 
+dependencies. When it is done, you will have an `assets` folder in your current
+working directory. This folder contains all of your asset dependencies, which
+should be importable by path specifiers.
+
+You can also specify the output folder where assets should be generated to:
+
+```
+$ empathy install -a my/asset/dir
+```
+
+You can restrict which packages are installed as assets. To select specific
+packages to convert, use `--only`:
+
+```
+$ empathy install --only \@polymer/polymer
+```
+
+You can also select specific packages to ignore using `--ignore`:
+
+```
+$ empathy install --ignore express
+```
+
+For extreme mindfulness, add empathy to your `postinstall` step:
+
+```js
+{
+  "name": "cool-dogs-package",
+  "version": "1.0.0",
+  // ...
+  "scripts": {
+    "postinstall": "empathy install"

```

Comment:
I don't think this is quite correct for a few reasons:
 * If `cool-dogs-package` was to be published, `empathy install` would be run if someone ran `npm install cool-dog-package`; I assume this is not expected based on the convention that that only names should be published.
 * If I were developing `cool-dogs-package` and I ran `npm install --save button-component`, this `postinstall` script would not be run, which is precisely when I want it be run.

Instead, as above, I would probably prefer this to another script and probably placed in the `build` script. Obviously, this means optimizations would need to be made (caching etc) and a reduction in the output of empathy.